CMakeLists.txt

```diff
@@ -128,6 +128,40 @@ if (BUILD_SDF)
   add_subdirectory(doc)
 endif(BUILD_SDF)
 
+########################################
+# Python interfaces
+option(USE_SYSTEM_PATHS_FOR_PYTHON_INSTALLATION
+      "Install python modules in standard system paths in the system"
+      OFF)
+
+option(USE_DIST_PACKAGES_FOR_PYTHON
+      "Use dist-packages instead of site-package to install python modules"
+      OFF)
+
+include(IgnPython)
```

Comment:
there is already some code to find python up at lines 60-90; we should unify that code with this

Reply:
https://github.com/ignitionrobotics/sdformat/pull/932/commits/05545f844df895c4c1157e773061a4e5dcdb8148